### PR TITLE
Fix RemoteBitfield findFirst edge case 

### PR DESCRIPTION
**Pull request by HDegroote**

### CODE REVIEW

File: test/remote-bitfield.js [hunk 1]
```diff
@@ -27,3 +30,41 @@ test('set last bits in segment and findFirst', function (t) {
   t.is(b.findFirst(false, 32766), 32768)
   t.is(b.findFirst(false, 32767), 32768)
 })
+
+test('remote congituous length consistency (remote-bitfield findFirst edge case)', async function (t) {
```
Comment (HDegroote) [inline test/remote-bitfield.js:34]: I added the test to the remote bitfield tests because it detects an issue with the bitfield logic, but since it's using the replicator API it could also live in `test/remote-length.js`
